.travis.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Please tell me that's not why the Travis builds were failing...

Reply:
haha. No, it wasn't. (It was because this was switched to an open source library) :-P